Makefile

```diff
@@ -2,6 +2,7 @@ all: test install
 
 install:
 	go install ./...
+	go get github.com/stretchr/testify
```

Comment:
Since this is only a test dependency could you move this to the test target?  I'll look into adding proper dependency management in the future as well but okay to leave in the Makefile for now.

Reply:
I usually used glide, works pretty ok, but it's up to you.